utils/api.electron.js

```diff
@@ -0,0 +1,19 @@
+import electron from 'electron'
```

Comment:
Please add more descriptive comments about this file and for each method what it does and returns.